.changeset/rare-bats-notice.md

```diff
@@ -0,0 +1,5 @@
+---
+"docs": major
```

Comment:
```suggestion
"docs": patch
```

For context, if this is major/minor it causes the next version bump of Vitessce to be major/minor respectively

Reply:
good to know, thanks!